gcp-jobs/filings-notebook-report/tests/unit/test_job.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+import os
+import shutil
+import psycopg2
+import pytest
+import ast
+from notebookreport import processnotebooks
+
+
+def test_connection_failed():
+    status = False
+    try:
+        connection = psycopg2.connect(user=os.getenv('FAKE_PG_USER', ''),
+                                      password=os.getenv('FAKE_PG_PASSWORD', ''),
+                                      host=os.getenv('FAKE_PG_HOST', ''),
+                                      port=os.getenv('FAKE_PG_PORT', '5432'),
+                                      database=os.getenv('FAKE_PG_DB_NAME', ''))
+
+        cursor = connection.cursor()
+        status = True
+    except Exception:
+        status = False
+    finally:
+        assert status == False
+
+
+def test_connection_succeed():
+    status = False
+    try:
+        connection = psycopg2.connect(user=os.getenv('PG_USER', ''),
+                                      password=os.getenv('PG_PASSWORD', ''),
+                                      host=os.getenv('PG_HOST', ''),
+                                      port=os.getenv('PG_PORT', '5432'),
+                                      database=os.getenv('PG_DB_NAME', ''))
+        cursor = connection.cursor()
+        status = True
+    except Exception:
+        status = False
+    finally:
+        assert status == True
+
+
+test_filings_monthly_data = [
+    ("daily"), ("monthly"),
+]
+
+
+@pytest.mark.parametrize("report_type", test_filings_monthly_data)
+def test_filings_monthly_notebook_report(report_type):
+    data_dir = os.path.join(os.getcwd(), r'data/')
+    if not os.path.exists(data_dir):
+        os.makedirs(data_dir)
+
+    status = processnotebooks(report_type, data_dir)
```

Comment:
I guess they are expecting you to setup the db ahead of time?? 🤦 

This is unfortunate, and I think it may be worth pulling in the model and doing a bit of setup / reworking how the notebooks connect to the db a bit -- we can discuss. This is out of scope, but I think it'll add a lot of value for us to have at least 1 real unit test thats properly setup

Reply:
Yea thats why I asked about a dump yesterday the test doesn't work without a preexisting db